src/Microsoft.DotNet.Build.Tasks/PackageFiles/Roslyn.Common.props

```diff
@@ -1,37 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <RoslynVersion>3.0.0-beta4-final</RoslynVersion>
-    <RoslynPackageName>Microsoft.Net.Compilers</RoslynPackageName>
+    <RoslynVersion>3.1.0-beta1-19172-05</RoslynVersion>
+    <RoslynPackageName>Microsoft.Net.Compilers.Toolset</RoslynPackageName>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <RoslynPropsFile Condition="'$(RoslynPropsFile)' == '' and '$(RunningOnCore)' != 'true'">$(BuildToolsTaskDir)roslyn/build/Microsoft.Net.Compilers.props</RoslynPropsFile>
-  </PropertyGroup>
-
-  <!--
-    On Unix we always use a version of Roslyn we restore from NuGet and we have to work around some known issues.
-  -->
-  <PropertyGroup Condition="'$(RoslynPropsFile)' == '' and '$(RunningOnCore)' == 'true'">
-    <RoslynPackageName>Microsoft.NETCore.Compilers</RoslynPackageName>
+  <PropertyGroup Condition="'$(RoslynPropsFile)' == ''">
     <RoslynPackageDir>$(PackagesDir)/$(RoslynPackageName.ToLower())/$(RoslynVersion)/</RoslynPackageDir>
     <RoslynPropsFile>$(RoslynPackageDir)build/$(RoslynPackageName).props</RoslynPropsFile>
 
     <!--
       Portable PDBs are now supported in Linux and OSX with .Net Core MSBuild.
     -->
-    <DebugType>Portable</DebugType>
+    <DebugType Condition="'$(RunningOnCore)' == 'true'">Portable</DebugType>
   </PropertyGroup>
 
-
   <!-- If we're not using the compiler server, set ToolPath/Exe to direct to
        the exes in this package -->
   <PropertyGroup Condition="'$(UseRoslynCompilers)' != 'false' and '$(RoslynIncompatibleMsbuildVersion)' == 'true'">
-    <CscToolPath Condition="'$(RunningOnCore)' == 'true'">$(RoslynPackageDir)tools</CscToolPath>
-    <CscToolPath Condition="'$(RunningOnCore)' != 'true'">$(BuildToolsTaskDir)roslyn/tools</CscToolPath>
+    <CscToolPath>$(_RoslynTasksDirectory)</CscToolPath>
```

Comment:
I actually couldn't find anything that depends on this, so maybe we can just remove it (especially given we don't look to have an equivalent in Arcade)?

Reply:
Remove this entire `PropertyGroup`. It's just referring to items we don't need anymore.

Reply:
Done. Will leave the thread as "unresolved" for visibility, however.